suite-common/wallet-core/src/discovery/discoveryThunks.ts

```diff
@@ -594,8 +594,15 @@ export const startDiscoveryThunk = createThunk(
                     // do nothing. error will be handled in lower block
                 }
             }
+            console.log('==== discovery result.payload.error', result.payload.error);
 
-            if (result.payload.error && device.connected) {
+            if (
+                // todo: I am not sure if this code block makes sense. do we know why it is here?
+                result.payload.error &&
+                // not when another application stole this device
+                result.payload.error !== 'Device is used in another window' &&
+                device.connected
+            ) {
```

Comment:
I did a minimal change here to exclude just one case of possible errors where we know that we don't want this block to execute. 
But  I am still struggling to understand in what other cases this block might be desirable.